booklet/theme.json

```diff
@@ -0,0 +1,231 @@
+{
+	"version": 2,
+	"customTemplates": [
+		{
+			"name": "blank",
+			"title": "Blank",
+			"postTypes": [
+				"page",
+				"post"
+			]
+		}
+	],
+	"settings": {
+		"appearanceTools": true,
+		"color": {
+			"palette": [
+				{
```

Comment:
If I switch to this theme from a theme that has `primary`, `secondardy` and `tertiary` colors defined then my patterns will break because this theme doesn't define them. Maybe this doesn't matter....

Reply:
Yeah, I'm not totally sure we _have_ to keep these. I guess it's part of the ongoing discussion over in [29568](http://href.li?https://github.com/WordPress/gutenberg/issues/29568).

I've added in a few more colors for now. They're unused in the theme at the moment, but they'll come in handy for patterns in the future. https://github.com/Automattic/themes/pull/5213/commits/d7e9c4e4fdac70227a29b908166a50f35f81884f